apps/shopify/shopify.go

```diff
@@ -0,0 +1,25 @@
+// Package shopify provides details for the Shopify applet.
+package shopify
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed shopify.star
+var source []byte
+
+// New creates a new instance of the Shopify applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "shopify",
```

Comment:
We have an internal Shopify app in the works, with a similar feature set. It's release is pending us setting up a public Shopify app, with their full authentication flow, so in the meantime I think this would be a wonderful addition! The internal app is however already hogging the `shopify` app ID, so this will need a different ID for this to get merged.

Reply:
Changed the name to Shopify Chart